docs/source/api/federation_resolvers.rst

```diff
@@ -139,6 +139,11 @@ Response Structure
 :ipAddress: The IP address or :abbr:`CIDR (Classless Inter-Domain Routing)`-notation subnet of the resolver - may be IPv4 or IPv6
 :type:      The :term:`Type` of the resolver
 
+	.. versionadded:: 1.4
```

Comment:
should this change as 1.4 was already cut.

Also out of pure curiosity is there a reason why this was added?

Reply:
To make it consistent with `GET` response bodies. Now it returns _extra_ information, but not _different_ information.

Reply:
Is a 1.5 upgrade added yet, or do I have to do that in this PR too? I know there's a PR that would bump the version, dunno if it was merged yet.

Reply:
Actually, in this case, this message is correct. This is me adding to the docs for an already-rewritten endpoint. `/federation_resolvers` does this on master; i.e. in 1.4

Reply:
ahhh good catch!